superagent/superagent.d.ts

```diff
@@ -77,6 +77,7 @@ declare module "superagent" {
       notAcceptable: boolean;
       notFound: boolean;
       forbidden: boolean;
+      xhr: XMLHttpRequest;
```

Comment:
Thank you for your answer. 
You helped me to point my error => "dom" was missing in my tsconfig lib entry.